location-permissions/src/commonMain/kotlin/LocationPermission.kt

```diff
@@ -15,30 +15,30 @@
 
  */
 
-package com.splendo.kaluga.permissions.location
+package com.splendo.kaluga.locationpermissions
```

Comment:
Strongly suggest keeping the original package names.

Reply:
I wanted to align it with folder names, but okay, I'll change it

Reply:
yeah as we discussed we probably want to move the folders into a subfolder at some point, so it's aligned with that. Also two words smashed together in lowercase 💥 is not so pretty.